README.md

```diff
@@ -57,7 +57,7 @@ ElSelect('p').AddLorem({
 ```
 *After select the element you can add your lorem text with **AddLorem()** and the options*
 
-Another Functions :
+There are two other Functions :
```

Comment:
in future may will add some other functions ! i think this is not very interesting for the this section title

Reply:
No problem but 'Another' is related to just one other thing, If there are more things, we use 'other'. you should solve that, you can do it in another way.

Reply:
Thanks for your suggestion